.gitignore

```diff
@@ -10,3 +10,4 @@ Build/
 venv/
 guitest.log
 stale_outputs_checked
+**/wrapper/
```

Comment:
Why was this added?

Reply:
Because we keep getting wrappers in `jetbrains-ultimate` after running tests :( . It won't exclude already included files, the only thing it does it make it require an additional flag to update the gradle wrapper.

Reply:
Where it is putting wrappers? I have not seen that lcoally

Reply:
`jetbrains-ulatimate/wrapper`

Reply:
Reverted `.gitignore` change